Confuser.Core/Helpers/MutationHelper.cs

```diff
@@ -102,14 +102,10 @@ public static class MutationHelper {
 								pendingInstructions.Enqueue(method.Body.Instructions[argIndex]);
 						}
 
-						var firstArgIndex = initialLoadInstructions.Select(method.Body.Instructions.IndexOf).Min();
-						Instruction[] arg = method.Body.Instructions.Skip(firstArgIndex).Take(i - firstArgIndex).ToArray();
-						for (int j = 0; j < arg.Length; j++)
-							method.Body.Instructions.RemoveAt(firstArgIndex);
-						method.Body.Instructions.RemoveAt(firstArgIndex);
-						arg = repl(arg);
-						for (int j = arg.Length - 1; j >= 0; j--)
-							method.Body.Instructions.Insert(firstArgIndex, arg[j]);
+						var firstArgIndex = initialLoadInstructions.Min(instruction =>
```

Comment:
Any reason for not simply passing the reference to the method on here?
```suggestion
						var firstArgIndex = initialLoadInstructions.Min(method.Body.Instructions.IndexOf);
```